lib/presentation/crowdaction/crowdaction_comments/parts/comment_item.dart

```diff
@@ -0,0 +1,53 @@
+part of '../crowdaction_comments_page.dart';
+
+/// The comment body that includes the commenter's avatar and name
+/// along with the comment content.
+///
+/// This also has [CommentActions] at the bottom.
+///
+/// Currently, this takes a [CrowdactionCommentDto] variable [comment],
+/// that has predicted comment values, however these will change in future.
+///
+/// The [CrowdactionCommentDto] will also be converted to a [CrowdactionComment]
+/// that is yet to be created.
```

Comment:
If you want I can send you the exact response we can expect from the API.

Reply:
That would be great